### PR TITLE
[identity] Users, Teams, Orgs

### PR DESCRIPTION
**Pull request by theBeginner86**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: content/en/cloud/identity/users/_index.md [hunk 1]
```diff
@@ -1,12 +1,7 @@
 ---
 title: Users
-description: >
-  A short lead description about this section page. Text here can also be **bold** or _italic_ and can even be split over multiple paragraphs.
-date: 2017-01-05
+description: Smallest atomic unit of tenancy in the system.
```
Comment (leecalcote) [inline content/en/cloud/identity/users/_index.md:3]: Orgs are the unit of tenancy.

Comment (theBeginner86) [reply content/en/cloud/identity/users/_index.md:3]: yes, I was looking to describe Users as the atomic unit of tenancy in the system.

Looks like it's making the sentence ambiguous. I'll improve this area. 👍 
Thanks for feedback.

Comment (leecalcote) [inline content/en/cloud/identity/users/_index.md:3]:
```suggestion
description: Each user account represents an individual collaborator. Individual user accounts exist beyond the bounds of organizations.
```

File: content/en/cloud/identity/users/_index.md [hunk 2]
```diff
@@ -1,12 +1,7 @@
...
+weight: 3
 ---
 
-{{% pageinfo %}}
-Organizations, Teams, Users
-{{% /pageinfo %}}
-
-YOu can create any number of these.
\ No newline at end of file
+Users are the smallest unit of tenancy in the Layer5 Cloud. They are the entities that can be authenticated and authorized to access resources in the system. Users can be created and managed by the "Provider Admins" or by the users themselves.
```
Comment (leecalcote) [inline content/en/cloud/identity/users/_index.md:7]: Bear in mind who the audience is here. While Provider Admins may read this documentation, initially and largely, the audience is the end-user.

**Pull request merged**